lib/collectionspace/converter/core/acquisition.rb

```diff
@@ -9,73 +9,100 @@ def convert
           end
         end
 
-        def self.map(xml, attributes)
-          CSXML.add xml, 'acquisitionReferenceNumber', attributes["acquisitionreferencenumber"]
-
-          CSXML.add_group xml, 'accessionDate', {
-            "dateDisplayDate" => CSDTP.parse(attributes["accessiondategroup"]).display_date,
-            "dateEarliestScalarValue" => CSDTP.parse(attributes["accessiondategroup"]).earliest_scalar,
-            "dateLatestScalarValue" => CSDTP.parse(attributes["accessiondategroup"]).latest_scalar,
+       def self.map(xml, attributes)
+         pairs = {
```

Comment:
@Paulmulonzia - not sure why some of the initial converter tool work defined `pairs` and `repeats` as separate functions of the module. 

It's better to just set them up as variables like this. Then you don't have to call them in any special way to pass them as arguments to the further data preparation methods (`add_pairs`, etc)

Reply:
Noted, using variables going forward.